azure/microservices/database-setup.yml

```diff
@@ -43,6 +43,8 @@ spec:
     envFrom:
     - configMapRef:
         name: tb-node-db-config
+    - configMapRef:
```

Comment:
I guess it would be nice to have a complete set of variables and secrets that tb-node has. You never know when developers decide to work on Kafka or Zookeeper during the update. 
```
    envFrom:
      - configMapRef:
          name: tb-node-db-config
      - configMapRef:
          name: tb-node-env-config
      - configMapRef:
          name: tb-kafka-config
      - configMapRef:
          name: tb-cache-config
```